library/xwiki

```diff
@@ -2,20 +2,20 @@ Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
 # Stable
-Tags: 17, 17.10, 17.10.0, 17-mysql-tomcat, 17.10-mysql-tomcat, 17.10.0-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 17, 17.10, 17.10.1, 17-mysql-tomcat, 17.10-mysql-tomcat, 17.10.1-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 Architectures: amd64, arm64v8
 Directory: 17/mysql-tomcat
-GitCommit: b1dc0934aa1761ef397459f7dbf866480e9fc6c9
+GitCommit: e30edad4fe23b9c1559685a8a04c92ae316fd1c8
```

Comment:
The build tests can't find this [commit](https://github.com/xwiki/xwiki-docker/commit/e30edad4fe23b9c1559685a8a04c92ae316fd1c8) in `xwiki-contrib/docker-xwiki`. Perhaps it is not pushed up yet?